### DISCUSSION TITLE
切换百度网盘账号，浏览器不变，

### DISCUSSION THREAD

**Discussion by minazuki2022** [syhyz1990/baiduyun#252] [Q&A]
原来百度账号A用chrome可以用脚本，但是切换成百度账号B之后，同样的浏览器，同样的设置，脚本却不能用了，不知道哪里出错？

**Comment by minazuki2022** [comment]
忘了提，然后再切回去百度账号A又可以用了

**Reply by National777** [reply]
我也是这样不过不能用的百度网盘号都是这两年注册的号容量只有105g，但是比较老的号就能用，运行脚本时直接没有链接
![屏幕截图 2024-04-20 185053](https://github.com/syhyz1990/baiduyun/assets/114747659/1f3be2cf-27f8-4714-adb3-445ccc39fb5f)
